util/app-config/src/app_config.rs

```diff
@@ -401,45 +401,7 @@ mod tests {
                 miner_config.chain.spec,
                 PathBuf::from("specs/integration.toml")
             );
-            assert_eq!(miner_config.miner.rpc_url, "http://127.0.0.1:7000/");
-        }
-    }
-
-    #[cfg(all(unix, target_pointer_width = "64"))]
```

Comment:
Wrong merge?

Reply:
The test is added by @xxuejie when porting to Windows.

Reply:
sorry, resolved auto rebase error, please review again.